### DISCUSSION TITLE
Cannot delete an organizer ("pretixbase.LogEntry") [second organizer on a fresh pretix installation]

### DISCUSSION THREAD

**Discussion by Wikinaut** [pretix/pretix#3913] [Q&A]
* pretix 2024.1.0

I wish to delete an organizer, but always receive the message

> "The organizer could not be deleted as some constraints (e.g. data created by plug-ins) do not allow it. The following database models still contain data that cannot be deleted automatically: pretixbase.LogEntry"

> "Dieser Veranstalter konnte nicht gelöscht werden, da einige Bedingungen (z.B. von Plugins erstellte Daten) es nicht erlauben. Die folgenden Datenbankmodelle enthalten noch Daten, die nicht automatisch gelöscht werden können: pretixbase.LogEntry"

Please can anyone give me an advice?

**Comment by raphaelm** [comment]
pretix is not made for things to be deleted it got even harder recently, not sure if the best way to fix this is removing the delete feature or if we actually want to fix it.

**Reply by MagicLike** [reply]
Hey I just ran into this exact error myself - are there any updates on fixing this?

**Reply by raphaelm** [reply]
Not yet, sorry

**Comment by SIP-Online** [comment]
Working on an Odoo and Pretix Bridge, I ran into this problem.
I think it is good to keep the deletion of an organizer.
Concerning the solution, I put this code on my side before deleting the organizer :
```
        qs = LogEntry.objects.filter(organizer = EXAMPLE_ID_ORGANIZER)

        for logentry in qs:
            super(LogEntry, logentry).delete()
```

Why go to super() ?
Because your Model LogEntry contains an _inherit with a raise inside 😞 : https://github.com/pretix/pretix/blob/be6aae8577fe0448024086ae35c6fae697d2f4b7/src/pretix/base/models/log.py#L250

So I go directly to a delete() of the Django Model.
